app/views/tasks/index.html.erb

```diff
@@ -33,12 +40,22 @@
         <%= button_to "Update Task", "/modify/#{task.id}", method: :get %>
       </td>
       <td>
+         <%= button_to "Mark Done!", "/tasks/mark_complete/#{task.id}",  method: :patch, data: { confirm: "Mark this task as Complete?" } %>
+      </td>
+      <td>
         <%= button_to "Delete Task", "/delete_confirmation/#{task.id}", method: :get %>
       </td>
-
+      <td>
+        <% unless task.person.nil? %>
```

Comment:
Nice use of the `.nil?` method - you could DRY this up even more and reduce these three lines to one